FirebaseAppDistribution/Sources/FIRAppDistribution.m

```diff
@@ -346,6 +347,27 @@ - (void)handleOauthDiscoveryCompletion:(OIDServiceConfiguration *_Nullable)confi
                                                      callback:processAuthState];
 }
 
+- (BOOL)tryInitializeAuthState {
+  NSLog(@"Initializing auth state");
+
+  if (self.authState) {
+    NSLog(@"Auth state already initialized.");
+    return true;
+  }
+
+  NSError *authRetrievalError;
+  self.authState = [self.authPersistence retrieveAuthState:&authRetrievalError];
+  // TODO (schnecle): replace NSLog statement with FIRLogger log statement
+  if (authRetrievalError) {
+    NSLog(@"Error retrieving tester auth token");
```

Comment:
I still think that this log statement is misleading because we will error anytime we don't find the authState. We only want this to tell us there is an error if we know you have signed in. This is only good enough to tell us that we were unable to retrieve the authState but not whether or not we should have been able to.

Reply:
Fair point. I'll fix it to not be an error log.